src/runtime-pmi2.c

```diff
@@ -0,0 +1,210 @@
+/* -*- C -*-
+ *
+ * Copyright 2011 Sandia Corporation. Under the terms of Contract
+ * DE-AC04-94AL85000 with Sandia Corporation, the U.S.  Government
+ * retains certain rights in this software.
+ * 
+ * This file is part of the Sandia OpenSHMEM software package. For license
+ * information, see the LICENSE file in the top level directory of the
+ * distribution.
+ *
+ *
+ * Run-time support for the built-in runtime that is part of the
+ * shared memory implementation Portals
+ */
+
+#include "config.h"
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#if defined(PMI_SLURM)
```

Comment:
you aren't defining this in your m4 like we were in orte...m4 and this causes build issues with the slurm+pmi2 combo